src/screencast/pipewire_screencast.c

```diff
@@ -26,6 +26,44 @@ static void writeFrameData(void *pwFramePointer, void *wlrFramePointer,
 	return;
 }
 
+static inline struct spa_pod *build_format(struct spa_pod_builder *b, uint32_t format,
```

Comment:
Can't we use `enum spa_video_format` for the format here? That would make function calls slightly less error-prone.

Reply:
sure

Reply:
done